python/pacemaker/_cts/corosync.py

```diff
@@ -149,10 +153,15 @@ def start(self, kill_first=False, timeout=10):
                                                         self.cluster_name, localname())
         logfile = corosync_log_file(BuildOptions.COROSYNC_CONFIG_FILE)
 
+        if self._env["have_systemd"]:
+            cmd = ["systemctl", "start", "corosync.service"]
+        else:
+            cmd = ["corosync"]
+
```

Comment:
It's not worth straightening this out in this PR, but if you really want to be annoyed, check out `pacemaker/_cts/tests/remotedriver.py` which already knows how to use the Environment to start corosync.